src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -569,4 +569,23 @@ public String getThriftProtocolClass() {
     public String getMetricsCollectorClass() {
         return mProperties.getString("secor.monitoring.metrics.collector.class");
     }
+    
+    public Map<String, String> getPropertyMapForPrefix(String prefix) {
```

Comment:
I'll add the Java doc. This is ORC schema and in case of invalid format, it'll not allow secor application to start.